pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -5255,7 +5255,7 @@ def analyze_fit_results(self):
         if self.leakage_qbname is not None:
             # get leakage
             if self.get_param_value('classified_ro', False):
-                leakage = self.leakage_values[0::2] - self.leakage_values[1::2]
+                leakage = self.leakage_values[0::2]
```

Comment:
@stephlazar  why is this change required? Isn't the goal to look at the leakage introduced by the gate, rather than the absolute leakage? The absolute leakage, we can get from the single qubit readout calibration (as we only do single qubit gates before the cphase gate)